[bug] zwavejs 5.0.0 doesn't run

<!--
  🚨🚨🚨 STOP! STOP! STOP! 🚨🚨🚨
  Before opening an issue, please read and follow these steps:

  1. Are you using HomeAssistant?
  If yes, please open your issue at https://github.com/home-assistant/core/issues
  UNLESS a developer told you to come here.

  2. Check the troubleshooting section if your problem is described there:
  https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/bug_report

  3. Check the changelog if your problem was already fixed recently.
  https://github.com/zwave-js/zwavejs2mqtt/blob/master/CHANGELOG.md
  We cannot provide support if you are not using the latest version.

  4. Make sure to provide the necessary information, as described here:
  https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/bug_report
  
  If you are using HomeAssistant, this is how you do it:
  Home Assistant -> settings -> Integrations -> Z-Wave JS -> Configure -> Create dump -> zip the json file and post it here.

  🙏🏻🙏🏻🙏🏻 Thanks, now onto your issue:
-->

### Version

**Checklist:**

- [X] I am **not** using HomeAssistant. **Or:** a developer has told me to come here. - i opened at HomeAssistant and they instructed me to open issue here
- [X] I have checked the troubleshooting section and my problem is **not** described there.
- [X] I have read the changelog problem was **not** mentioned there.

Build/Run method

- [ ] Docker
- [ ] PKG
- [ ] Snap package
- [X] Manually built (git clone - yarn install - yarn run build )

zwavejs2mqtt version: 5.0.0
zwave-js version: 7.7.3

### Describe the bug

zwavejs2mqtt doesn't start

Error in logfile:
```
Writing the device index to disk failed: File or directory '/**/zwavejs2mqtt/node_modules/@zwave-js/config/config/devices/index.json' was not included into executable at compilation stage. Please recompile adding it as asset or script
```

Results of `systemctl status zwavejs2mqtt`:
● zwavejs2mqtt.service - zwavejs2mqtt server
   Loaded: loaded (/etc/systemd/system/zwavejs2mqtt.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Thu 2021-06-10 13:52:27 MDT; 11s ago
  Process: 45269 ExecStart=/home/jhass/zwavejs2mqtt/zwavejs2mqtt (code=exited, status=1/FAILURE)
 Main PID: 45269 (code=exited, status=1/FAILURE)

Jun 10 13:52:27 hass zwavejs2mqtt[45269]:     at Object.Module._extensions..node (internal/modules/cjs/loader.js:1248:18)
Jun 10 13:52:27 hass zwavejs2mqtt[45269]:     at Module.load (internal/modules/cjs/loader.js:1047:32)
Jun 10 13:52:27 hass zwavejs2mqtt[45269]:     at Function.Module._load (internal/modules/cjs/loader.js:935:14)
Jun 10 13:52:27 hass zwavejs2mqtt[45269]:     at Module.require (internal/modules/cjs/loader.js:1087:19)
Jun 10 13:52:27 hass zwavejs2mqtt[45269]:     at Module.require (pkg/prelude/bootstrap.js:1338:31)
Jun 10 13:52:27 hass zwavejs2mqtt[45269]:     at require (internal/modules/cjs/helpers.js:73:18)
Jun 10 13:52:27 hass zwavejs2mqtt[45269]:     at bindings (/snapshot/zwavejs2mqtt/node_modules/bindings/bindings.js:112:48)
Jun 10 13:52:27 hass zwavejs2mqtt[45269]:     at Object.<anonymous> (/snapshot/zwavejs2mqtt/node_modules/@serialport/bindings/lib/linux.js:2:3
Jun 10 13:52:27 hass systemd[1]: zwavejs2mqtt.service: Main process exited, code=exited, status=1/FAILURE
Jun 10 13:52:27 hass systemd[1]: zwavejs2mqtt.service: Failed with result 'exit-code'.
```

### To Reproduce

Steps to reproduce the behavior:

1. stop zwavejs2mqtt
2. upgrade to zwavejs2mqtt v5.0.0 by unzipping zwavejs2mqtt-v5.0.0.zip
3. start zwavejs2mqtt (i have a systemd unit file, so i simply do systemctl start zwavejs2mqtt

### Expected behavior

zwavejs2mqtt should be running when i check via `pgrep zwavejs2mqtt`
it isn't running, however.

### Additional context

Reinstalling zwavejs2mqtt v 4.5.2 runs fine.

- zwavejs2mqtt is installed on my hass machine via git clone from the repository
- first i stop zwavejs2mqtt
- i simply unzip and overwrite bindings.node and zwavejs2mqtt executable
- then i restart by `systemctl start zwavejs2mqtt`
- contents of my systemd unit file:

[Unit]
Description=zwavejs2mqtt server
After=network-online.target

[Service]
Type=simple
User=jhass
WorkingDirectory=/home/jhass/zwavejs2mqtt
ExecStart=/home/jhass/zwavejs2mqtt/zwavejs2mqtt

[Install]
WantedBy=multi-user.target

@robertsLando Maybe it's the same issue. I did a container recreate to update to release 5.00. After that upgrade the containers starts successful but ZwaveJs2Mqtt isn't running. I saw the following error in de log in the Docker container

`Error: Cannot find module '/usr/src/app/bin/www'

    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)

    at Function.Module._load (node:internal/modules/cjs/loader:769:27)

    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)

    at node:internal/main/run_main_module:17:47 {

  code: 'MODULE_NOT_FOUND',

  requireStack: []

}

node:internal/modules/cjs/loader:927

  throw err;

  ^



Error: Cannot find module '/usr/src/app/bin/www'

    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)

    at Function.Module._load (node:internal/modules/cjs/loader:769:27)

    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)

    at node:internal/main/run_main_module:17:47 {

  code: 'MODULE_NOT_FOUND',

  requireStack: []

}

node:internal/modules/cjs/loader:927

  throw err;

  ^



Error: Cannot find module '/usr/src/app/bin/www'

    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)

    at Function.Module._load (node:internal/modules/cjs/loader:769:27)

    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)

    at node:internal/main/run_main_module:17:47 {

  code: 'MODULE_NOT_FOUND',

  requireStack: []

}`

--- a/package.json
+++ b/package.json
@@ -106,7 +106,7 @@
     "mocha": "^8.4.0",
     "node-notifier": "^9.0.1",
     "npm-run-all": "^4.1.5",
-    "pkg": "^4.5.1",
+    "pkg": "4.4.9",
     "portfinder": "^1.0.28",
     "proxyquire": "^2.1.3",
     "release-it": "^14.8.0",

